### ISSUE TITLE
Using configured alias in logs

### ISSUE THREAD

**Issue by JensErat** [fluent/fluent-bit#1923]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Right now, each plugin instance can have an [`Alias` configured that is used in metrics output](https://docs.fluentbit.io/manual/v/1.3/configuration/monitoring#configuring-aliases). It seems to me that this is ignored for fluent-bit logs. For example, we have two outputs configured like this:

    [OUTPUT]
        Name          forward
        Alias         user
    <snip>
    
    [OUTPUT]
        Name          forward
        Alias         platform
    <snip>

    [2020/01/29 13:16:43] [error] [out_fw] no upstream connections available
    [2020/01/29 13:16:44] [ warn] [engine] Task cannot be retried: task_id=2 thread_id=2 output=forward.0

The warning at least includes the `forward.0` string, the error misses this information entirely. I'd love to have something like

    [2020/01/29 13:16:43] [error] [out_fw] no upstream connections available: task_id=2 thread_id=2 output=forward.user
    [2020/01/29 13:16:44] [ warn] [engine] Task cannot be retried: task_id=2 thread_id=2 output=forward.user

**Describe alternatives you've considered**

**Additional context**

If there is agreement, I might be able to provide the code if I find some time, but I don't want to promise too much.

**Comment by edsiper** [comment]
I totally agree. This feature is mostly about implement a logging API per plugin type and based on it display a better logging message.

+1

**Comment by danielpodwysocki-gini** [comment]
Any plans for this?

Is there another discussion/issue around it?
